src/data/accessibility-quiz.ts

```diff
@@ -1,4 +1,15 @@
 const accessibilityQuiz = [
+  {
+    Question:
+      "Why is it important to use semantic elements when writing HTML?",
+    Answer: "Semantic code clearly describes the code's meaning to both the browser and the developer",
+    Distractor1: "Semantic code is prettier.",
+    Distractor2: "Semantic code decreases site readability.",
+    Distractor3: "Semantic code can improve comprehension for hearing-impaired individuals.",
+    Explanation:
+      "Authors are strongly encouraged to view the div element as an element of last resort, for when no other element is suitable. Use of more appropriate elements instead of the div element leads to better accessibility for readers and easier maintainability for authors.",
```

Comment:
we want to avoid using direct quotes from documentation 👍 
https://html.spec.whatwg.org/multipage/grouping-content.html#the-div-element

```suggestion
      "Semantic HTML helps describe the meaning behind the content whereas generic HTML elements like divs and spans hold no special meaning. It is always encouraged to use semantic HTML when possible to help with accessibility.",
```